pyvista/utilities/helpers.py

```diff
@@ -1458,3 +1459,56 @@ def cubemap(path='', prefix='', ext='.jpg'):
         texture.SetInputDataObject(i, flip.GetOutput())
 
     return texture
+
+
+def set_default_active_vectors(mesh: 'pyvista.DataSet') -> None:
+    """Set a default vectors on mesh, if not already set.
+
+    If an active vector already exists, no changes are made.
+
+    If an active vectors does not exist, it checks for possibly cell
+    or point arrays with shape ``(n, 3)``.  If only one exists, then
+    it is set as the active vectors.  If none or more than one exists,
+    then a ValueError is raised.
+
+    Parameters
+    ----------
+    mesh : pyvista.DataSet
+        Dataset to set default active vectors.
+
+     Raises
+    ------
+    MissingDataError
+        If no vector-like arrays exist.
+    AmbiguousDataError
+        If more than one vector-like arrays exist.
+
+    """
+    if mesh.active_vectors_name is not None:
+        return
+
+    point_data = mesh.point_data
+    cell_data = mesh.cell_data
+
+    possible_vectors_point = [
+        name for name, value in point_data.items() if value.ndim == 2 and value.shape[1] == 3
+    ]
+    possible_vectors_cell = [
+        name for name, value in cell_data.items() if value.ndim == 2 and value.shape[1] == 3
+    ]
+
+    possible_vectors = possible_vectors_point + possible_vectors_cell
+    n_possible_vectors = len(possible_vectors_point) + len(possible_vectors_cell)
+
+    if n_possible_vectors == 1:
+        if len(possible_vectors_point) == 1:
+            mesh.set_active_vectors(possible_vectors_point[0], preference='point')
+        else:
+            mesh.set_active_vectors(possible_vectors_cell[0], preference='cell')
+    elif n_possible_vectors < 1:
+        raise MissingDataError("No vector-like data available.")
+    elif n_possible_vectors > 1:
+        raise AmbiguousDataError(
+            f"Multiple vector-like data available: {possible_vectors}.\n"
+            "Set one as active using DataSet.set_active_vectors(name)"
```

Comment:
I wonder how confusing this would be if there's a single point vectors and a single cell vectors with the same name (might actually be more common than expected). We'd print the same name twice, and the `set_active_vectors` call would probably also need a `preference` kwarg to distinguish the two.

But I have no idea how to make this more descriptive for this case while keeping it concise.

Reply:
We don't need to set a preference to call this function because we have, so far, taken the decision to raise an error if there is more than one vector-like array.

It would be most complete to print the vector-like arrays separately for points and cells.  But we don't do this for usage like `mesh.array_names` so I don't know.

Reply:
With the preference I just meant the recommendation in the error message: "Set one as active using `DataSet.set_active_vectors(name)`". But again it would be cumbersome to try and add the preference distinction to the message (to the effect of "also don't forget to choose the right preference if you have a name clash").

Reply:
We could print cell and point vectors separately and then use `DataSet.set_active_vectors(name, preference=type)` something like this.  This gives the user two pointers without having to be super explicit.